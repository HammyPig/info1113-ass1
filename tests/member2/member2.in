ADD MEMBER gibbeh
ADD MEMBER im sam
ADD MEMBER and im carly
ADD MEMBER and this is icarly
meMber 199999
member 100001
MEMBER 100002
memBER 100003
MEMBER 100000