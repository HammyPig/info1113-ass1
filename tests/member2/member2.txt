outputs member info (possibly including spacing), or if member is not found