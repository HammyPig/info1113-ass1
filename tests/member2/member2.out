user: Success.

user: Success.

user: Success.

user: Success.

user: No such member in system.

user: 100001: im sam

user: 100002: and im carly

user: 100003: and this is icarly

user: 100000: gibbeh

user: Ending Library process.
